src/ESA-PL-weak-copyleft-2.4.xml

```diff
@@ -0,0 +1,648 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="ESA-PL-weak-copyleft-2.4"
+   name="European Space Agency Public License – v2.4
+   – Weak Copyleft (Type 2)" listVersionAdded="">

```

Comment:
Value of the attribute (name) has to be on single line.
Please add value to listVersionAdded. Set it to to next milestone version, i.e., 3.28.0